extensions/big-data-cluster/src/extension.ts

```diff
@@ -83,3 +89,21 @@ function deleteControllerInternal(treeDataProvider: ControllerTreeDataProvider,
 		treeDataProvider.saveControllers();
 	}
 }
+
+
+
+/**
+ * Throttles actions to avoid bug where on clicking in tree, action gets called twice
```

Comment:
This is handling a recent bug since the last vscode merge. If we don't do this, we'll get a dialog shown behind the current one